### DISCUSSION TITLE
memory bank size?

### DISCUSSION THREAD

**Discussion by shi-yan** [gpuweb/gpuweb#4400] [Q&A]
how to know the bank size in order to achieve bank conflict free compute shader?

Thanks

**Comment by Kangz** [comment]
There isn't a way in WebGPU to know the bank size of workgroup memory. However if you really absolutely need to know, you can try matching the GPUAdapterInfo's vendor + architecture (when available) to known bank sizes based on hardware vendor whitepapers.
